app/crud/categories_update.php

```diff
@@ -12,11 +12,11 @@
     $slug = $_POST['slug'];
     $title = $_POST['title'];
 
-    $categories = Category::findById($id);
-    $categories->setCompetitionId($competition_id);
-    $categories->setSlug($slug);
-    $categories->setTitle($title);
-    $result = $categories->update();
+    $category = Category::findById($id);
+    $category->setCompetitionId($competition_id);
+    $category->setSlug($slug);
+    $category->setTitle($title);
+    $result = $category->update();
```

Comment:
The `update()` method does not return a value.

```suggestion
    $category->update();
```

Please do this on the [**`$event->update()`**](https://github.com/benjamineke/sportsfest-litmusda/blob/dev/app/crud/events_update.php#L19) too.